test/js/bun/test/jest-extended.test.js

```diff
@@ -99,6 +99,40 @@ describe("jest-extended", () => {
 
   // toBeOneOf('toSatisfy()')
 
+  test("toBeOneOf()", () => {
+    expect(1).toBeOneOf([1, 2, 3]);
+    expect(2).toBeOneOf([1, 2, 3]);
+    expect(3).toBeOneOf([1, 2, 3]);
+    expect(4).not.toBeOneOf([1, 2, 3]);
+    expect("a").toBeOneOf(["a", "b", "c"]);
+    expect("b").toBeOneOf(["a", "b", "c"]);
+    expect("c").toBeOneOf(["a", "b", "c"]);
+    expect("d").not.toBeOneOf(["a", "b", "c"]);
+    expect(true).toBeOneOf([true, false]);
+    expect(false).toBeOneOf([true, false]);
+    expect(null).toBeOneOf([null, undefined]);
+    expect(undefined).toBeOneOf([null, undefined]);
+    const abc = {};
+    expect({}).not.toBeOneOf([{}, []]);
```

Comment:
vscode put suggestion on wrong line, but this and line 118 need to be updated after adding deepEquals